### PR TITLE
chore(module): add chakra-ui module

### LINKED ISSUE #12: dead repos
- https://github.com/nuxt-community/material-icons-module (alternative https://github.com/anteriovieira/nuxt-material-design-icons ?)
- https://github.com/jmblog/nuxt-netlify-http2-server-push (support for Netlify http2 server push has been dropped -  https://community.netlify.com/t/why-was-http-2-server-push-support-dropped/14770/4)
- https://github.com/nuxt-community/modules/tree/master/packages/bulma
- https://github.com/nuxt-community/modules/tree/master/packages/plugin-utils

### PR DESCRIPTION
**Pull request by Geminii**
Resolves #12 

Suggested by @Atinux to add chakra-ui module

### CODE REVIEW

**Comment by Atinux** [comment]
It looks there is some issue regarding this PR, do you mind opening another one @Geminii ?

**Comment by Geminii** [comment]
> It looks there is some issue regarding this PR, do you mind opening another one @Geminii ?

Yes sorry i forgot to rebase my upstream ^^
Looks good now 😀

File: modules/chakra-ui.yml
Comment (Atinux) [inline modules/chakra-ui.yml:2]: Description is wrong @Geminii

File: modules/chakra-ui.yml
Comment (Atinux) [inline modules/chakra-ui.yml:7]:
```suggestion
website: 'https://vue.chakra-ui.com/with-nuxt'
```

File: modules/chakra-ui.yml
Comment (Atinux) [inline modules/chakra-ui.yml:6]:
```suggestion
github: 'https://github.com/chakra-ui/chakra-ui-vue/tree/develop/packages/chakra-ui-nuxt'
```

File: modules/chakra-ui.yml
Comment (Atinux) [inline modules/chakra-ui.yml:12]:
```suggestion
  - name: Jonathan Bakebwa
```

File: modules/chakra-ui.yml
Comment (Atinux) [inline modules/chakra-ui.yml:13]:
```suggestion
    github: codebender828
```

File: modules/chakra-ui.yml
Comment (Atinux) [inline modules/chakra-ui.yml:14]:
```suggestion
    avatar: 'https://avatars0.githubusercontent.com/u/21237954?v=4'
```

**Review by Atinux — CHANGES_REQUESTED** [review]
See my suggestions

File: modules/chakra-ui.yml
Comment (Geminii) [reply modules/chakra-ui.yml]: We got this error into another sync module probably due to the description package. 
I will be careful for next integration

**Comment by Geminii** [comment]
> See my suggestions

I change all of it and update the avatar too.

**Review by Atinux — APPROVED** [review]
Approved, waiting for @pi0 approval

**Review by pi0 — APPROVED** [review]
LGTM. Thanks!

**Pull request merged**
